### PR TITLE
Add in test for C16

### PR DESCRIPTION
**Pull request by tien-han**
# Description
This PR seeks to address test case C16 in Issue #962.
 
There are two changes in this PR:
1. Wrote a test for C16 within `readingsCompareMeterFlow.js`, covering a 7 day shift ending on 2022-10-31 17:00:00 for 15 minute intervals and flow units.
2. Added `thing_unit` unit, conversion, and meter definitions to `readingUtils.js` and standardized usage across tests.

Partially Addresses Issue #962

## Type of change
N/A
- [ ] Note merging this changes the database configuration.
- [ ] This change requires a documentation update

## Checklist
- [x] I have followed the [OED pull request](https://openenergydashboard.github.io/developer/pr.html) ideas
- [ ] I have removed text in ( ) from the issue request (NOTE: not relevant?)
- [x] You acknowledge that every person contributing to this work has signed the [OED Contributing License Agreement](https://openenergydashboard.github.io/developer/cla.html) and each author is listed in the Description section. 

## Limitations
N/A

## Images of tests passing post-standardization of "thing" objects:
1. readingsBarMeterFlow.js
image
3. readingsLineGroupFlow.js
image
4. readingsLineMeterFlow.js
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ See the full licensing agreement [here](LICENSE.txt)
 
 ## Contributions ##
 
-We welcome others to contribute to this project by writing code for submission or collaborating with us. Before contributing, please sign our [Contributor License Agreement](https://openenergydashboard.github.io/developer/cla.html). Web pages with [information for developers](https://openenergydashboard.github.io/developer/) is available. If you have any questions or concerns feel free to at engage@OpenEnergyDashboard.org.
+We welcome others to contribute to this project by writing code for submission or collaborating with us. Before contributing, please sign our [Contributor License Agreement](https://openenergydashboard.org/developer/cla/). Web pages with [information for developers](https://openenergydashboard.github.io/developer/) is available. If you have any questions or concerns feel free to at engage@OpenEnergyDashboard.org.
```
Comment (tien-han) [inline README.md:37]: The current link doesn't work so I updated this to the formal link for the CLA. Happy to remove this change if preferred.

**Comment by tien-han** [comment]
Please note that this test will fail until PR #1362 has been merged.

File: src/server/util/readingsUtils.js [hunk 2]
```diff
@@ -277,6 +277,62 @@ const groupDatakWh = [
     }
 ];
 
+// These are the 2D arrays for units and conversions to feed into the database
```
Comment (huss) [inline src/server/util/readingsUtils.js]: While it is generally a good idea to encapsulate reused code as you have done here, OED made the decision that all but kWh related units would not be in this util file and each test would need to define them. I'm sorry to ask this but can you please remove the Thing definitions from this file and remove it from the test files. Please let me know if you have questions/thoughts about this.

**Review by huss — CHANGES_REQUESTED** [review]
Thanks to @tien-han for their first contribution to OED. Overall, this works as expected. I made one comment about a change to conform to how OED desires the code (sorry if that was not clear earlier).

Also, I appreciate you finding and taking the time to fix up the links in the contributing file. I missed that on the large update to the website files.

File: src/server/util/readingsUtils.js (hunk 2 above)
Comment (tien-han) [reply src/server/util/readingsUtils.js]: No worries, thank you so much for reviewing the code and the heads up! We'll make the change as soon as possible.

**Comment by huss** [comment]
I'm just noting I merged development since my recent merge of C15 caused a conflict.

**Review by huss — APPROVED** [review]
Thanks to @tien-han for the updated code. Review and testing found it works as expected. Congratulations on your first accepted contribution to OED.

**Pull request merged**